code/modules/mob/dead/observer/observer.dm

```diff
@@ -296,7 +296,11 @@ This is the proc mobs get to turn into a ghost. Forked from ghostize due to comp
 	set name = "Ghost"
 	set desc = "Relinquish your life and enter the land of the dead."
 
-	if(stat == DEAD)
+	if(src.health < 0 && src.health > -95.0) //crit people
+		succumb()
+		sleep(5)
+		ghostize(1)
+	else if(stat == DEAD)
 		ghostize(1)
 	else
```

Comment:
Intigracy what the fuck are you talking about there is no way for both that `sleep()` and this code block to execute in the same call
I think I know what you actually mean, but you're explaining it incorrectly